loaders/ecg_loader_bin.py

```diff
@@ -0,0 +1,115 @@
+import os
+import numpy as np
+import torch
+import pandas as pd
+from loaders.gdd import GoogleDriveDownloader
+import pathlib
+import shutil
+
+
+GOOGLE_FILE_ID = '17Rd4YpGwssSpk4xZAT5AyYskjvs95dAY'
+ZIP_NAME = 'ecg_zip.zip'
+
+
+idx2label = {
+    0: 'Normal',
+    1: 'Abnormal'
+}
+
+
+# class names and short hand class names
+classnames = ['Normal', 'Abnormal']
+sh_classnames = ['Normal', 'Abnormal']
+class_importance = np.array([1, 2])
+data_sets = ['mitbih_train.csv', 'mitbih_test.csv']
+
+
+class ECGLoader_bin(torch.utils.data.Dataset):
```

Comment:
Can't we somehow integrate that with the ECGLoader? It's copying code a bit and makes the masking to be duplicated as well